index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="style.css">
+      <title>Document</title>
+   </head>
+
+   <body>
+      <div class="container">
+         <h1>Awesome Books</h1>
+         <div class="bookList">
+            
+         </div>
+         <div>
+            <form action="" id="form-action">
+               <input type="text" name="text" placeholder="title" class="title"><br><br>
+               <input type="text" name="text" placeholder="author" class="author"><br><br>
+               <button id="btn-add" type="submit">Add</button>
+            </form>
```

Comment:
- [ ] Please add required validation input to prevent the creation of empty books 👇🏻 

![image](https://github.com/MariamOsman11/Awesome-Books/assets/79058364/70222ffe-91cd-420e-a691-427f364bff9e)